doc/source/whatsnew/v0.13.1.rst

```diff
@@ -89,10 +86,13 @@ Output Formatting Enhancements
 - The ``ArrayFormatter`` for ``datetime`` and ``timedelta64`` now intelligently
   limit precision based on the values in the array (:issue:`3401`)
 
-  Previously output might look like:
-
-  .. code-block:: python
+  Previously output might look like:::
 
+      >>> df = pd.DataFrame([pd.Timestamp('20010101'),
+                            pd.Timestamp('20040601')], columns=['age'])
+      >>> df['today'] = pd.Timestamp('20130419')
+      >>> df['diff'] = df['today'] - df['age']
+      >>> df
```

Comment:
I'm happy with that, but I think there are `.. code-block:: text` or something like that, which would show in a block but won't raise the error.